packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -1576,8 +1974,8 @@ class Scaffold extends StatefulWidget {
 
 /// State for a [Scaffold].
 ///
-/// Can display [SnackBar]s and [BottomSheet]s. Retrieve a [ScaffoldState] from
-/// the current [BuildContext] using [Scaffold.of].
+/// Can display [BottomSheet]s. Retrieve a [ScaffoldState] from the current
```

Comment:
This seems to sell ScaffoldState short. What about drawers?

Reply:
It did not mention drawers before? I can certainly add it.